File: Back/src/main/java/com/defitech/GestUni/dto/EtudiantDetailsDto.java
package com.defitech.GestUni.dto;

import lombok.Data;

@Data
public class EtudiantDetailsDto {
    private String nom;
    private String prenom;
    private String parcours;
    private String filiere;
    private String theme;
    private String typeDocument;
    private String annee;
    private String documentUrl;
    private String professeurNom;
    private String professeurPrenom;
}

File: notefronte/src/app-chahib/app/models/EtudiantDetailsDto.ts
export class EtudiantDetailsDto {
    nom: string;
    prenom: string;
    parcours: string;
    filiere: string;
    theme: string;
    typeDocument: string;
    annee: string;  // Ajout de l'année
    documentUrl: string;
    professeurNom: string;
    professeurPrenom: string;
      constructor() {
        this.nom = '';
        this.prenom = '';
        this.parcours = '';
        this.filiere = '';
        this.theme = '';
        this.typeDocument = '';
        this.annee = '';
        this.documentUrl = '';
        this.professeurNom = '';
        this.professeurPrenom = '';
      }
    
  }
  